config/development.env

```diff
@@ -1,3 +1,3 @@
 APP_ENV=development
-REPORTS_DIR=/data/reports/
+REPORTS_DIR=s3://ooni-public/sanitised
```

Comment:
My understanding is that this PR should stop using `s3` in favor of the pipeline. Shouldn't this mean that we should stop using `s3://` URLs as part of that?